draft-ietf-tls-external-psk-guidance.md

```diff
@@ -106,34 +107,36 @@ informative:
 
 --- abstract
 
-This document provides usage guidance for external Pre-Shared Keys (PSKs) in TLS.
+This document provides usage guidance for external Pre-Shared Keys (PSKs)
+in Transport Layer Security (TLS) version 1.3 as defined in RFC 8446.
 It lists TLS security properties provided by PSKs under certain assumptions and
-demonstrates how violations of these assumptions lead to attacks. This document
-also discusses PSK use cases, provisioning processes, and TLS stack implementation
+demonstrates how violations of these assumptions lead to attacks. It
+discusses PSK use cases, provisioning processes, and TLS stack implementation
 support in the context of these assumptions. It provides advice for applications
-in various use cases to help meet these assumptions. Privacy and security properties
-not provided by PSKs are also included.
+in various use cases to help meet these assumptions. It also lists the privacy
+and security properties that are not provided by TLS when external PSKs are used.
 
 --- middle
 
 # Introduction
 
-There are many resources that provide guidance for password generation and verification aimed towards improving security.
-However, there is no such equivalent for external Pre-Shared Keys (PSKs) in TLS. This document aims to reduce
-that gap. It lists TLS security properties provided by PSKs under certain assumptions and demonstrates
-how violations of these assumptions lead to attacks. This document also discusses PSK use
-cases, provisioning processes, and TLS stack implementation support in the context of these
-assumptions. It provides advice for applications in various use cases to help meet these
-assumptions.
+This document provides guidance on the use of external Pre-Shared Keys (PSKs)
```

Comment:
Can we revert this particular change? (cc @cbartle891)

Reply:
This addresses a comment.  It you want to keep it, it belongs later in the Introduction.

Reply:
That's fine then. Can you please move it to some place later in the intro? (I just don't want to lose it.)

Reply:
I put it in the second paragraph in the intro.